publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,86 @@
+    And they see the response body has a field "detail" with the value "The request identifier is invalid: {malformed identifier}"
+
+  Scenario: The Owner publishes an already published Publication
+    Given the Owner has published the Publication
+    But the Publication is still not indexed
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{identifier}/publish
+    Then they receive a response with status code 202
+    And they see that the response Content-type is "application/json"
+    And they see that the response Location header is the URI to the Publication
+    And they see the response body has a field "message" with the value "Publication is being published. This may take a while."
+
+  Scenario: The Owner publishes an already published and indexed Publication
+    Given the Owner has published the Publication
+    And the Publication is indexed
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{identifier}/publish
+    Then they receive a response with status code 200
+    And they see that the response Content-type is "application/json"
+    And they see that the response Location header is the URI to the Publication
+    And they see the response body has a field "message" with the value "Publication is published."
+
+  Scenario: The Owner checks that the Publication is published
+    Given the Owner has published the Publication
+    And the Publication is indexed
+    When they set the Accept header to "application/json"
+    And they request GET /publication/{identifier}
+    Then they receive a response with status code 200
+    And they see that the response Content-type is "application/json"
+    And they see that the response body is a Publication JSON object
+    And they see that the response body has a field "status" with the value "PUBLISHED"
+    And they see that the response body has a field "indexedDate" with a value "{datetime}"
+
+  Scenario: The Owner publishes an invalid Publication
+    Given the Owner wants to publish an invalid Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{identifier}/publish
+    Then they receive a response with status code 422
+    And they see that the response Content-Type header is "application/problem+json"
+    And they see that the response body is a problem.json object
+    And they see the response body has a field "title" with the value "Unprocessable Entity"
+    And they see the response body has a field "status" with the value "422"
+    And they see the response body has a field "detail" with the value "Publication is missing data required for publishing."
```

Comment:
I think we should give an explicit response containing a validation report, noting what needs to be fixed…I guess it is enough for now to say it contains the string "The publication cannot be published because the following fields are not populated" 
And the missing fields are listed